### PR TITLE
[broker] Add command to delete a cluster's metadata from ZK

### PR DESCRIPTION
**Pull request by BewareMyPower**
### Motivation

When we share the same ZK and BK cluster with multiple broker clusters, if a cluster was removed, its metadata in ZK should also be removed.

### Modifications

- Add a `PulsarClusterMetadataTeardown` class to delete the relative nodes from ZK;
- Wrap the class to `bin/pulsar` script.

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataTeardown.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                return;
+            }
+        } catch (Exception e) {
+            jcommander.usage();
+            throw e;
+        }
+
+        ZooKeeper localZk = initZk(arguments.zookeeper, arguments.zkSessionTimeoutMillis);
+
+        deleteZkNodeRecursively(localZk, "/bookies");
```
Comment (jiazhai) [inline pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataTeardown.java:75]: some of these parameters should be read from conf file?

Comment (BewareMyPower) [reply pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataTeardown.java:75]: `/managed-ledgers` and `/namespaces` are hard coded in `PulsarClusterMetadataSetup`, `/schemas` is hard coded in `BookkeeperSchemaStorage`,  `/loadbalance` is hard coded in `broker.loadbalance` package. The left ZK nodes I can't find the definition, but I don't find it configurable. If they're all hard coded, IMO, reading them from conf file is unnecessary.

Comment (jiazhai) [reply pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataTeardown.java:75]: get it

**Review by jiazhai — APPROVED** [review]

**Comment by wolfstudy** [comment]
Move this change to 2.6.2, because the #8244 depends on it.

**Pull request merged**